### PR TITLE
Add az module instruction

### PR DESCRIPTION
**Pull request by juhlee-microsoft**
Az module is required
Fixed a typo

### CODE REVIEW

**Comment by juhlee-microsoft** [comment]
The REAME text update does not need the change test result porting.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ LISAv2 includes below test suite categories:
 
 1. Download Web Platform Installer from [here](http://go.microsoft.com/fwlink/p/?linkid=320376&clcid=0x409)
 2. Start Web Platform Installer and select Azure PowerShell (required 6.3.0 or above) and proceed for Azure PowerShell Installation.
+3. Install the Azure Powershell Az module [here](https://docs.microsoft.com/bs-cyrl-ba/powershell/azure/install-az-ps?view=azps-2.5.0)
```
Comment (LiliDeng) [inline README.md]: I open the URL, seems some words are not English.

Comment (juhlee-microsoft) [reply README.md]: Fixed to US-en page.

**Comment by juhlee-microsoft** [comment]
@simonxiaoss  please review and sign off this PR.

**Pull request merged**